test/go-tests/test_utils.go

```diff
@@ -34,6 +34,7 @@ const (
 	KeptnSpecVersion      = "0.2.0"
 	KeptnNamespaceEnvVar  = "KEPTN_NAMESPACE"
 	DefaultKeptnNamespace = "keptn"
+	KeptnNamespacePrefix  = "KEPTN_NAMESPACE_PREFIX"
```

Comment:
Do we need this additional environment variable or could this be replaced by `KEPTN_NAMESPACE` - since both are set to the same value in `integration_test.yml` anyway?

Reply:
I will remove the prefix variable in favor of just prefixing everything with the keptn test namespace